filebeat/input/log/input.go

```diff
@@ -558,6 +572,8 @@ func (p *Input) scan() {
 				continue
 			}
 			if err != nil {
+				p.healthy = false
+				p.updateStatus(status.Degraded, fmt.Sprintf(harvesterErrMsg, newState.Source, err))
```

Comment:
Looks like this is the only place where we report a degraded state. There are no other potential places?

Reply:
@rdner this `updateStatus` captures errors during harvester creation/setup/registry errors.

Reply:
I'll check if I missed any potential places.

Reply:
@rdner I've added it at a couple of more placed.